### PR TITLE
Fixed GoLang Style issues using go simple and go lint tools

### LINKED ISSUE #102: Review and Correct for golang Coding Standards.
(Sorry, not filling out the form for this one.)
Cursory first glance at the k2cli golang code shows a number of violated golang standards.
Naming conventions,
best practices,
formatting.

Tools that should always be run:
golint, gosimple, go fmt, go vet.

### PR DESCRIPTION
**Pull request by alejandroEsc**
**What this PR does / why we need it**: Brings up to par style issues found by running the tools go simple and go lint.

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #102 

**Special notes for your reviewer**:

**Release note**:

```NONE
```

### CODE REVIEW

**Comment by alejandroEsc** [comment]
@mikeln related to the issue you brought up in https://github.com/samsung-cnct/k2cli/issues/102, please review when you have a chance.

File: cmd/bindata.go [hunk 1]
```diff
@@ -182,9 +182,10 @@ type bintree struct {
 	Func     func() (*asset, error)
 	Children map[string]*bintree
 }
+
 var _bintree = &bintree{nil, map[string]*bintree{
-	"data": &bintree{nil, map[string]*bintree{
-		"kraken_store": &bintree{dataKraken_store, map[string]*bintree{}},
+	"data": {nil, map[string]*bintree{
```
[the remark this answers is not in the document]
Comment (alejandroEsc) [reply cmd/bindata.go]: For Jenkins, yeah that is the plan, but I had to test the commands out, currently the jenkins `simple .` command does not work as intended. 

Take a look at https://github.com/samsung-cnct/k2cli/pull/157

I think you will like what I did there, and would be a nice addition to the jenkins file, if I can integrate it correctly.

As for bindata.go, correct, that was me failing to ignore a file change, sigh. Ill revert it. That should not be an issue moving forward.

**Comment by alejandroEsc** [comment]
@coffeepac i think my other message was dropped when the PR was changed. Please take a look at https://github.com/samsung-cnct/k2cli/pull/157 i plan to make the jenkin changes there or in another PR.

**Comment by mikeln** [comment]
I'm not sure we should stop jenkins approval on any, or all output results from go: vet, simple, lint, fmt.
My experience has been: 
# go fmt  should part of you editor (or even checkin) save (it is standard practice to include this in vim go setup).  Any changes it makes should just be done and included.
# lint, simple, vet - usually are valid, sometimes are not...can depend on what you are attempting to accomplish.

I guess my point, I've seen recommendations from these tools that may not be valid.  We will need a way to allow such things to occur.

**Comment by alejandroEsc** [comment]
@mikeln if we wish this ticket doesnt add anything to jenkins, but we can make another pr to remove what is there @coffeepac to confirm. In the meantime i suggest we ask absolutely that developers run these tools and make a judgement call if they should adhere to the advice. My other pr does create such a tool via `make verify`

**Comment by alejandroEsc** [comment]
@mikeln the redesign is not in a ticket, its just moving things from one helper file to another because it was found it made more logical sense. This happened as part of of using the tools as #102 mentions to do.

**Comment by mikeln** [comment]
sure... I think I still have brain mush from CKA tests...

**Pull request merged**